### DISCUSSION TITLE
Ender3 V2 with Sprite Pro firmware

### DISCUSSION THREAD

**Discussion by Hansd007** [mriscoc/Ender3V2S1#567] [Q&A]
Hello, 

I have an Ender3 V2 with 4.2.7 MB, CR touch, Sprite Pro and dual Z axis. What firmware should I use?

Thx in advance and thx for your hard work! 🙏

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Special_Configurations/releases/tag/T13
